getting-started/scaling-out.md

```diff
@@ -30,72 +29,64 @@ and removed using [`delete_data_node`][delete_data_node].
 
 Note that:
 
-* Data nodes are added as objects locally and connect to an
-  existing PostgreSQL instance.
+* A data node is represented on the access node by a local object that
+  contains the configuration needed to connect to a database on a
+  PostgreSQL instance. The [`add_data_node`][add_data_node] command is
+  used to create this object.
 
-* You should already have a running PostgreSQL server on the data node host.
+* You should already have a running PostgreSQL server on an instance
+  that will host a data node. The data node's database will
+  be created when executing the [`add_data_node`][add_data_node]
+  command on the access node and should _not_ exist prior to adding
+  the data node.
 
-* Ensure that the data node has password authentication enabled
-  in their `pg_hba.conf` files for any non-superusers.
+* PostgreSQL instances that will act as data nodes are assumed to
+  contain the same roles and permissions as the access node
+  instance. Currently, such roles and permissions need to be created
+  manually, although there is a [utility command][distributed_exec]
+  that can be used to create roles and permissions across data nodes.
+
+* A data node needs
+  [`max_prepared_transactions`][max_prepared_transactions]
+  set to a value greater than zero.
 
 When creating the data node, you should:
 
-* Provide a name to use when referring to the data node from
-  this database.
+* Run `add_data_node` as a superuser that can authenticate with the
+  data node instance. This can be done by setting up either password
+  or certificate [authentication][data-node-authentication].
 
-* Provide the host where the hypertable partition for the distributed
-  hypertable should be stored.
+* Provide a name to use when referring to the data node from
+  the access node database.
 
-* Provide the remote password, which will be used by the current user
-  during access to the created remote data node.
+* Provide the host name, and optionally port, of the PostgreSQL
+  instance that will hold the data node.
 
-* Provide a bootstrap user and password, which is used to
-  create the data node. If the current user can be used, then
-  the boostrap user and password can be omitted.
+After creating the data node:
 
-* Ensure that the bootstrap user used for connecting to the data node
-  is a superuser.
-  This is necessary since
-  `add_data_node` expects to be able to create a
-  database on the remote data node and create
-  a TimescaleDB extension within it.
+* Ensure that non-superusers have `USAGE` privileges on the
+  `timescaledb_fdw` foreign data wrapper and any
+  data node objects they will use on the access node.
 
-* Ensure that the local user has `USAGE` privileges on the `timescaledb_fdw`
-  foreign data wrapper on the access node.
+* Ensure that each user of a distributed hypertable has a way to
+  [authenticate][data-node-authentication] with the data nodes they
```

Comment:
I'm assuming the `page-index.js` will have been modified to accommodate this page (from the other PR)?

Reply:
I assume that too :)